environments/dev-env/README.md

```diff
@@ -0,0 +1,395 @@
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+Run the following script to install the required package (Linux or MacOS ):
+
+```bash
+./install_req.sh
+```
+
+Alternatively, run the following command for MacOS platform:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/README.md)
+
+
+## Quickstart
+
+#### 1. Clone this repo:
+
+```bash
+git clone https://github.com/kcp-dev/edge-mc
+```
+
+#### 2. Change into the following directory path:
+
+```bash
+cd edge-mc/environments/dev-env
+```
+
+#### 3. Experiment with the kcp-edge 2023q1 PoC example scenarios:
+
+In this quickstart example we will deploy `stage 3` described in more details [here](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/example1/). It creates the following components:
+
+-  The infrastructure and the edge service provider workspace and lets that react to the inventory
+-  Two workloads, called “common” and “special” and in response to each EdgePlacement, the edge scheduler creates the corresponding SinglePlacementSlice object.
+-  The placement translator reacts to the EdgePlacement objects in the workload management workspaces
+
+```bash
+./install_edge-mc.sh --stage 3
+```
+
+NB: if you're using a macOS, you may see pop-us messages similar to the one below while deploying kcp-edge: 
+
+```bash
+  Do you want the application “kcp” to accept incoming network connections?
+```
+
+You can accept it or configure your firewall to suppress them by adding our kcp-edge components to the list of permitted apps.
+
+
+You should see an ouput similar to the one below:
+
+```bash
```

Comment:
The content here is not a bash script.
See https://github.com/github/linguist/blob/master/lib/linguist/languages.yml for the list of supported language tags.